e2e/Cargo.toml

```diff
@@ -0,0 +1,43 @@
+# You must change these to your own details.
+[package]
+name = "e2e"
+description = "E2E testing the source compiler"
+version = "0.1.0"
+authors = ["Mohammad Yousuf Minhaj Zia <yzia2000@gmail.com>"]
+categories = ["wasm"]
+readme = "README.md"
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+# If you uncomment this line, it will enable `wee_alloc`:
+#default = ["wee_alloc"]
+
+[dependencies]
+source-compiler = { path = "../source-compiler" }
+js-sys = "0.3.22"
```

Comment:
Any reason to fix a particular patch version?  The other crates in this project do not fix it, and readily accept a version bump.  Same for the rest of the dependencies in this file.